sudoexec/jstart

```diff
@@ -656,7 +656,13 @@ if [ ${vnet} -eq 1 ]; then
 		echo "${jname}" > ${tmpdir}/eth${eth_seq}.locked
 		${IFCONFIG_CMD} ${i} name eth${eth_seq} && ${IFCONFIG_CMD} eth${eth_seq} vnet ${jname}
 		if [ "${ip4_addr}" = "REALDHCP" ]; then
-			${JEXEC_CMD} ${jname} /sbin/dhclient eth${eth_seq}
+			. "${workdir}/jails-data/${jname}-data/etc/rc.conf.d/network" 2>/dev/null 1>&2
```

Comment:
mb test for file existance? 

[ -r ${workdir}/jails-data/${jname}-data/etc/rc.conf.d/network ] && . ${workdir}/jails-data/${jname}-data/etc/rc.conf.d/network

?